keyboards/theone/h75/ansi_tkl/keymaps/via/keymap.c

```diff
@@ -0,0 +1,109 @@
+	_______,	_______,	_______,										_______, 							_______,	MO(4),   	_______,	SIDE_MOD,	SIDE_VAD,   SIDE_HUI),
+// layer 5
+[5] = LAYOUT(
+	_______, 	_______,  	_______,  	_______, 	_______,  	_______,  	_______,  	_______,  	_______,  	_______, 	_______, 	_______, 	_______, 				_______,
+	_______, 	_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,   	_______,  	_______,   	_______,	_______, 	_______,	_______,
+	_______, 	_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,   	_______,  	_______,   	_______,	_______, 	_______,	_______,
+	_______,	_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,   	_______,  	_______,	_______, 	 			_______,	_______,
+	_______,				_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,	_______,	_______,				_______,	_______,
+	_______,	_______,	_______,										_______, 							_______,	MO(1),   	_______,	_______,	_______,    _______),
+// layer 6
+[6] = LAYOUT(
+	_______, 	_______,  	_______,  	_______, 	_______,  	_______,  	_______,  	_______,  	_______,  	_______, 	_______, 	_______, 	_______, 				_______,
+	_______, 	_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,   	_______,  	_______,   	_______,	_______, 	_______,	_______,
+	_______, 	_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,   	_______,  	_______,   	_______,	_______, 	_______,	_______,
+	_______,	_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,   	_______,  	_______,	_______, 	 			_______,	_______,
+	_______,				_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,	_______,	_______,				_______,	_______,
+	_______,	_______,	_______,										_______, 							_______,	MO(1),   	_______,	_______,	_______,    _______),
+// layer 7
+[7] = LAYOUT(
+	_______, 	_______,  	_______,  	_______, 	_______,  	_______,  	_______,  	_______,  	_______,  	_______, 	_______, 	_______, 	_______, 				_______,
+	_______, 	_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,   	_______,  	_______,   	_______,	_______, 	_______,	_______,
+	_______, 	_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,   	_______,  	_______,   	_______,	_______, 	_______,	_______,
+	_______,	_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,   	_______,  	_______,	_______, 	 			_______,	_______,
+	_______,				_______,   	_______,   	_______,  	_______,   	_______,   	_______,   	_______,   	_______,	_______,	_______,				_______,	_______,
+	_______,	_______,	_______,										_______, 							_______,	MO(1),   	_______,	_______,	_______,    _______),
```

Comment:
```suggestion
```

Reply:
I would like to reserve these layers for VIA

Reply:
you don't need to make filler layers for VIA, it's handled automatically

Reply:
Okay, thank you